src/components/InputForm.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react'
+import { useNote } from "../contexts/InputContext"
+import "./common.css"
+
+export default function InputForm() {
+    const { setNotes } = useNote()
+    const [title, setTitle] = useState("")
+    const [description, setDescription] = useState("")
+
+
+    const handleTitle = (value) => {
+        setTitle(value)
```

Comment:
Suggestion : while this is a simple event handler but it's good to make your state updates functional.